cpp/test/dist_sort_test.cpp

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
it would be nice to add separate test cases for important methods such as tableBinarySearch, MergeSortedTable.